ee/tools/puppet/fleetdm/lib/puppet/functions/fleetdm/preassign_profile.rb

```diff
@@ -14,7 +14,8 @@ def preassign_profile(profile_identifier, host_uuid, template, group = 'default'
     host = call_function('lookup', 'fleetdm::host')
     token = call_function('lookup', 'fleetdm::token')
     client = Puppet::Util::FleetClient.new(host, token)
-    response = client.preassign_profile(host_uuid, template, group)
+    run_identifier = "#{closure_scope.catalog.catalog_uuid}-#{Puppet[:node_name_value]}"
```

Comment:
Are there any concerns in Ruby around interpolating values that are undefined (or the Ruby equivalent)?

Reply:
I think we should be ok (not a Ruby expert either), but I know we'll always have a `closure_scope.catalog`, and as for `catalog_uuid`:

```
~/fleet $ irb

irb(main):003:0> "#{nil}-something"
=> "-something"
```